controllers/topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require('../models/topics.models');
+
+exports.getAllTopics = (req, res, next) => {
+    selectTopics().then((topics) => {
+        res.status(200).send({ topics });
+    }).catch(next)
```

Comment:
Not sure how this bit made it onto this particular pull request because I think it's a different task, might have been committed by accident 🤷